### PR TITLE
Intercepted url working

### PR DESCRIPTION
**Pull request by pasqLisena**
I solved issue https://github.com/vitvad/Access-Control-Allow-Origin/issues/22.

Now urls are persistent on local storage of Chrome.

### CODE REVIEW

**Comment by herkulano** [comment]
Thank you! :)

File: app/js/popup.js [hunk 1]
```diff
@@ -53,16 +53,26 @@ app.controller('PopupCtrl', ['$scope', function($scope) {
     });
   };
 
+  var updateUrlsInStorage = function(){
+	  console.log($scope.urls)
```
Comment (vitvad) [inline app/js/popup.js]: remove please debug code

**Comment by vitvad** [comment]
@pasqLisena, https://github.com/pasqLisena/Access-Control-Allow-Origin/blob/dev/app/js/popup.js#L42
here is code for proper save in localStorage
actually there is need only ```true``` param in watch or change to watchCollection not sure what will be more suitable here.

```
$scope.$watch('urls', function(newValue, oldValue) {
 			chrome.storage.local.set({'urls': $scope.urls});
 			chrome.extension.getBackgroundPage().reload();
		});
	},true);
```

File: app/js/popup.js (hunk 1 above)
Comment (pasqLisena) [inline app/js/popup.js]: Yes, I agree!

**Comment by pasqLisena** [comment]
It seems that the "true" works! I updated my version.

**Pull request merged**